modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -414,3 +415,21 @@ def test_virtual_partition_apply_not_returning_pandas_dataframe(
 
     apply_result = partition.apply(lambda df: 1).get()
     assert apply_result == 1
+
+
+@pytest.mark.skipif(
+    Engine.get() != "Ray",
+    reason="Only ray.wait() does not take duplicate object refs.",
```

Comment:
What happens for Dask?

Reply:
I ran the test on Dask, and it completed as expected.